src/pug/blog/report_11.pug

```diff
@@ -0,0 +1,55 @@
+extends _blog.pug
+
+block append subtitle
+  | Progress report #11
+
+block content
+  h2 Progress report #11
+  +news_header('21 April 2023', 'report_10', '')
+
+  h3 Firmware beta with alphanumeric input methods
+  p As a continuation of the research documented on Alphanumeric input #[+a('alphanumeric_input') blog entry], we now implemented an early version of some input methods (Glyph-stick, Daisywheel, 8-slot quick chat macros, and on-screen keyboard shortcut) and released it as #[+ax(org.fw_release+'0.89.0-beta') Firmware 0.89.0-beta], we hope to get plenty of feedback from the community to iterate over these.
```

Comment:
"research on alphanumeric input"

Reply:
Move below video, rephrase to "We hope to receive plenty of feedback" as separate sentence